docs/control/plugin/index.rst

```diff
@@ -34,7 +34,19 @@ Docker Remote API
 The Docker Remote API call which supports ``--volume-driver`` in the CLI is undocumented at time of writing.
 However, currently is it is specified by the ``VolumeDriver`` attribute of ``HostConfig`` under ``POST /containers/create``.
 Set the container ``Mount`` attributes according to the instructions in the :ref:`docker-plugin-cli` section.
-Full documentation for ``VolumeDriver`` will appear after the release of `Docker Engine 1.9`_.
+More information about the ``VolumeDriver`` can be found in the `Docker volumes plugin`_ documentation.
 
 .. _`Docker volumes plugin`: https://docs.docker.com/extend/plugins_volume/
-.. _`Docker Engine 1.9`: https://github.com/docker/docker/wiki/Engine-1.9.0
\ No newline at end of file
+
+Storage Profiles
+================
+
+To use :ref:`storage-profiles` with the Flocker plugin for Docker you will need Docker 1.9 or later.
+
+The Flocker plugin for Docker accepts a ``gold``, ``silver``, or ``bronze`` profile, via the volume driver's options parameter.
```

Comment:
Does the flocker plugin for docker restrict this? In practice, if the storage vendor designs different profiles than Gold, Silver and Bronze, they should be able to advertise others than Gold, Silver and Bronze. E.g. `MandatoryProfiles` implies you need these, but are not limited to them.
 (Mandatory)
https://github.com/ClusterHQ/flocker/blob/master/flocker/node/agents/blockdevice.py#L969

(EBS Based) In theory ebs could have others right? 
https://github.com/ClusterHQ/flocker/blob/master/flocker/node/agents/ebs.py#L111

So many we say, ```The Flocker plugin for Docker accepts a ``gold``, ``silver``, or ``bronze`` profile by default, via the volume driver's options parameter.``` or something